Scatter/Scatter/ViewController.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ViewController.swift
+//  Scatter
+//
+//  Created by Ryan Murphy on 5/22/19.
+//  Copyright © 2019 Ryan Murphy. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    var isScattered: Bool = false
+    var labels = [UILabel]()
+
+    var initialCenters: [CGPoint] = []
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        lambdaLogo.image = UIImage(named: "lambda_logo")
+        labels = [lLabel, aLabel, mLabel, bLabel, dlabel, secondALabel]
+        setLableCenter()
+      
+   }
+
+    @IBOutlet weak var lLabel: UILabel!
+    @IBOutlet weak var aLabel: UILabel!
+    @IBOutlet weak var mLabel: UILabel!
+    @IBOutlet weak var bLabel: UILabel!
+    @IBOutlet weak var dlabel: UILabel!
+    @IBOutlet weak var secondALabel: UILabel!
+    
+    @IBOutlet weak var lambdaLogo: UIImageView!
+    
+    
+    func setLableCenter() {
+        
+        let labels = [lLabel, aLabel, mLabel, bLabel, dlabel, secondALabel]
+        for label in labels {
+            guard let label = label else { return }
+            self.initialCenters.append(label.center)
+        }
+    }
+    func reArrangeLabels() {
+    let labels = [lLabel, aLabel, mLabel, bLabel, dlabel, secondALabel]
+    
+    UIView.animate(withDuration: 1, delay: 0, options: .curveEaseOut, animations: {
+    var count = 0
+    for label in labels {
+    guard let label = label else { return }
+    label.center = self.initialCenters[count]
+    label.transform = .identity
+    count += 1
+    }
+    })
+}
+    
+    
+    
+    static func randomColor() -> UIColor {
```

Comment:
Is there a reason why you made this function static? If you remove static you can just access it like this randomColor() instead of having to pass in the Viewcontroller.